internal/database/permission_sync_jobs.go

```diff
@@ -35,6 +37,7 @@ type PermissionSyncJobStore interface {
 	CreateRepoSyncJob(ctx context.Context, repo api.RepoID, opts PermissionSyncJobOpts) error
 
 	List(ctx context.Context, opts ListPermissionSyncJobOpts) ([]*PermissionSyncJob, error)
+	CancelSyncJob(ctx context.Context, opts CancelPermissionSyncJobOptions) error
```

Comment:
Maybe, I'm not sure:
```suggestion
	Cancel(ctx context.Context, opts CancelPermissionSyncJobOptions) error
```

Since the whole thing is called `PermissionSyncJobStore` already.